CHANGELOG.md

```diff
@@ -1,7 +1,8 @@
 # 6.4.2
 
 - Fixed: `selector-pseudo-class-case`, `selector-pseudo-class-no-unknown`, `selector-pseudo-element-case`, `selector-pseudo-element-no-unknown` rules now ignore SCSS variable interpolation.
-- Fixed: `selector-type-no-unknown` now ignores nested selectors and keyframe selectors.
+- Fixed: `selector-type-no-unknown` now ignores nested selectors, keyframe selectors, and placeholder selectors.
+- Fixed: `selector-type-case` now ignores placeholder selectors.
```

Comment:
Because 6.4.2 has been released, you'll need to create a new section in the Changelog titled `# Head` and add your fixes there.